src/bindings/jna/hello/src/main/java/HelloElektra.java

```diff
@@ -45,5 +45,15 @@ public static void main(String[] args) {
 
     Key n = Key.create("user:/weird\\/name///\\\\/is/\no/_\\\\problem");
     n.keyNameIterator().forEachRemaining(s -> System.out.println("itername: " + s));
+
+    Key cutpoint = Key.create("user:/cutpoint"),
+    	ka = Key.create("user:/cutpoint/hello", "hiback"),
+     	kb = Key.create("user:/cutpoint/hello2", "hellotoo"),
+     	kc = Key.create("user:/different/hello", "hellothere");
```

Comment:
Make more clear what this example shows:

```suggestion
     	kc = Key.create("user:/outside/cutpoint/hello", "hellothere");
```